plugin/keepeye.vim

```diff
@@ -5,6 +5,7 @@ let g:keepeye_system_bell = get(g:, 'keepeye_system_bell', v:false)
 let g:keepeye_system_notification = get(g:, 'keepeye_system_notification', v:false)
 let g:keepeye_timer = get(g:, 'keepeye_timer', 1500)
 let g:keepeye_bell_path = resolve(expand('<sfile>:h') . '/../media/bell.mp3')
+let g:keepeye_statusline_hi_group = get(g:, 'keepeye_statusline_hi_group', 0)
```

Comment:
I would rename to `g:keepeye_statusline_hl_group` (intead of `hi`) to match with the `hl-User` in the doc

Reply:
You're right. Renamed.